src/todo-func.js

```diff
@@ -0,0 +1,89 @@
+export const getTodos = () => JSON.parse(localStorage.getItem('todos')) || [];
+export function renderTodos() {
+  const todos = getTodos();
+
+  const todolist = todos
+    .sort((a, b) => a.index - b.index)
+    .map(
+      (todo) => `
+      <div class='todo-item' id="todo-item-${todo.index}">
+        <label>
+          <input type='checkbox' data-id="${todo.index}" class='todo-check' >
+          ${todo.description}
+        </label>
+        <div class='kebab' data-id="${todo.index}">
+          <i class="fa-solid fa-ellipsis-vertical"></i>
+        </div>
+        <i class="fa-solid fa-pen-to-square" data-id="${todo.index}"></i>
+        <i class="fa-solid fa-trash" data-id="${todo.index}"></i>
+      </div>
+    `,
+    )
+    .join('');
+
+  const task = document.querySelector('.tasks');
+  task.innerHTML = todolist;
+
+  const editBtn = document.querySelectorAll('.fa-pen-to-square');
+  editBtn.forEach((btn) => btn.addEventListener('click', handleEdit));// eslint-disable-line
+
+  const trashBtn = document.querySelectorAll('.fa-trash');
+  trashBtn.forEach((btn) => btn.addEventListener('click', handleDelete));// eslint-disable-line
```

Comment:
- [ ] great job making some of the changes. like the previous reviewer clearly stated, kindly remove the disable linters for clean code practices. This is because linters are important tools that help to identify potential errors and bugs in our code